docs/tutorials/getting_started.rst

```diff
@@ -2,207 +2,133 @@
+samples.
+
+Let's read some samples from the VDIF file::
+
+    >>> data = fh_vdif.read(3)
+    >>> data
+    array([[-1.      ,  1.      ,  1.      , -1.      , -1.      , -1.      ,
+             3.316505,  3.316505],
+           [-1.      ,  1.      , -1.      ,  1.      ,  1.      ,  1.      ,
+             3.316505,  3.316505],
+           [ 3.316505,  1.      , -1.      , -1.      ,  1.      ,  3.316505,
+            -3.316505,  3.316505]], dtype=float32)
+    >>> data.shape
+    (3, 8)
+
+Baseband decodes binary data into `~numpy.ndarray` objects.  Notice we
+input ``3``, and received an array of shape ``(3, 8)``; this is because
+there are 8 VDIF |threads|.  Threads and channels represent different
+|components| of the data such as polarizations or frequency sub-bands, and the
+collection of all components at one point in time is referred to as a
+:term:`complete sample`.  Baseband reads in units of complete samples,
+and works with sample rates in units of complete samples per second (including
+with the Mark 5B example above). Like an `~numpy.ndarray`, calling
+``fh_vdif.shape`` returns the shape of the entire dataset::
+
+    >>> fh_vdif.shape
     (40000, 8)
-    >>> fh.shape[0] # Number of complete samples.
-    40000
-    >>> fh.size
-    320000
-    >>> fh.ndim
-    2
 
-The shape of a single complete sample, including names indicating the meaning
-of shape dimensions, is retrievable using::
+The first axis represents time, and all additional axes represent the shape of
+a complete sample.  A labelled version of the complete sample shape is given
+by::
 
-    >>> fh.sample_shape
+    >>> fh_vdif.sample_shape
     SampleShape(nthread=8)
 
-By default, dimensions of length unity are |squeezed|, or removed from the
-sample shape.  To retain them, we can pass ``squeeze=False`` to
-`~baseband.vdif.open`:
+Baseband extracts basic properties and header metadata from opened files.
+Notably, the start and end times of the file are given by::
 
-    >>> fhns = vdif.open(SAMPLE_VDIF, 'rs', squeeze=False)
-    >>> fhns.sample_shape    # Sample shape now keeps channel dimension.
-    SampleShape(nthread=8, nchan=1)
-    >>> fhns.ndim            # fh.shape and fh.ndim also change with squeezing.
-    3
-    >>> d2 = fhns.read(12)
-    >>> d2.shape             # Decoded data has channel dimension.
-    (12, 8, 1)
-    >>> fhns.close()
+    >>> fh_vdif.start_time
+    <Time object: scale='utc' format='isot' value=2014-06-16T05:56:07.000000000>
+    >>> fh_vdif.stop_time
+    <Time object: scale='utc' format='isot' value=2014-06-16T05:56:07.001250000>
 
-Basic information about the file is obtained by either by ``fh.info`` or simply
-``fh`` itself::
+For an overview of the file, we an either print ``fh_vdif`` itself, or use the
+``info`` method::
 
-    >>> fh.info
+    >>> fh_vdif.info
     Stream information:
     start_time = 2014-06-16T05:56:07.000000000
     stop_time = 2014-06-16T05:56:07.001250000
```

Comment:
Alignment seems off below this, not sure why (maybe just an artifact of github).

Reply:
Seems to be fine on my end when looking at source and built docs.